python/paddle/fluid/dygraph/layers.py

```diff
@@ -132,8 +132,11 @@ def forward(self, input):
                 out = mylayer(x)
 
         """
-        # global setting
-        framework._dygraph_tracer().train_mode()
+        # global setting in dygraph
+        # NOTE(chenweihang): nn.Layer also can be used in static mode,
+        # but _dygraph_tracer() can not be called in static mode
+        if in_dygraph_mode():
+            framework._dygraph_tracer().train_mode()
```

Comment:
Actually, _dygraph_tracer().train_mode() can be removed.

Reply:
agree, and need to revert `dropout(is_test=None)` to `dropout(is_test=False)`

Reply:
for compatibility, we remove it in next PR is better, this PR need to be chery-pick into 2.0 to fix this bug